Editor/CreateNewPrefab.cs

```diff
@@ -91,7 +125,7 @@ static bool ValidateCreatePrefabOrVariant()
     [MenuItem("FishingCactus/Tools/Create New Prefab/New or Variant")]
     static void CreatePrefabOrVariant()
     {
-        CreatePrefabs(true, false);
+        CreatePrefabs( true, false, true );
```

Comment:
Can you explicitly write the parameters on those calls?

Reply:
Maybe you missed this comment?

Reply:
done.